### ISSUE TITLE
Revancify doesn't support armeabi-v7a 

### ISSUE THREAD

**Issue by PROBLEMCHYLD** [decipher3114/Revancify#212]
Even when creating on arm64 the package doesn't work when transfered to armeabi-v7a, can you guys fix this? It doesn't matter if I do online/offline.

**Comment by decipher3114** [comment]
The arm64 apk contains the architecture for that only. How do you expect it to work with the armeabi-v7a?

**Comment by PROBLEMCHYLD** [comment]
No, when I use com.google.android.youtube_18.25.39-1538522560_minAPI26(arm64-v8a,armeabi-v7a,x86,x86_64)(nodpi)_apkmirror.com.apk to create an apk using arm64 device it doesn't install on an armeabi-v7a. It only install arm64. If I use Revanced manager it works on all architecture

**Comment by decipher3114** [comment]
I still can't understand your problem. Could you please show me a screen recording or something, so that I can clearly get your issue.

**Comment by PROBLEMCHYLD** [comment]
I have a OnePlus 8T with Android 13 Oxygen OS. I installed Revancify and create the apk for YT and YTM. Once the apk package is complete, I installed on my OnePlus 8T Android 13 Oxygen OS. Those same packages doesn't work on my Android 10 Sky 55 phone. I know the architecture is different but when I use Revanced manager the apk packages work on all architectures. I'm not sure how else to explain.

**Comment by decipher3114** [comment]
Okay, Understood.
The YouTube app that is patched contains all arch, i.e, its universal. As you explained it, I can interpret you are using the Revanced Extended(inotia00) sources. The CLI tool by RVX has a feature called riplib that deletes the libs of the archs that are not needed (in your case, all other than arm64). If you want to use this app in armv7a device, you have to turn off this riplib feature. Go to Preferences -> riplibs (press spacebar to toogle) and then you can re-patch the app.


But, YouTube Music app is distibuted as separate arch by Google. The arm64 device only downloads and patch the arm64 version and it won't work in your armv7a device. To make it work, you need to download the app manually from [apkmirror.com](https://www.apkmirror.com/uploads/?appcategory=youtube-music) and then patch the app with revancify. In apps list, choose "From Apk file" option to proceed.

**Comment by PROBLEMCHYLD** [comment]
Thanks 👍👍
